mpqp/core/instruction/breakpoint.py

```diff
@@ -30,3 +34,13 @@ def __repr__(self) -> str:
             f"Breakpoint(targets={self.targets}, draw_circuit={self.draw_circuit},"
             f" enabled={self.enabled}, label={self.label})"
         )
+
+    def to_other_language(
```

Comment:
I would `pass` and filter them out at the circuit level

Reply:
done